hass-addon-sunsynk-dev/mqtt.py

```diff
@@ -63,6 +63,7 @@ class Entity:
     expire_after: int = attr.field(default=0)  # unavailable if not updated
     enabled_by_default: bool = attr.field(default=True)
     entity_category: str = attr.field(default="")
+    icon: str = attr.field(default="")
```

Comment:
Is the idea to send it in discovery info?

Reply:
Lets remove this as its not being used

Reply:
Ok, but not sure it was actually sent via MQTT though?
But lets add this in a new PR once we're done here

Reply:
Yep it was sent via MQTT. I can definitely put up another PR which just deals with icons.